tests/test_tools.py

```diff
@@ -189,3 +189,9 @@ def test_get_package_info(monkeypatch, inputs, expected):
     )
     actual_metadata = get_package_info(inputs[0], metadata=inputs[1])
     assert actual_metadata == expected
+
+
+def test_compute_mu_using_xraydb():
```

Comment:
we need a test for no density passed in.  Also for packing fraction passed in and also for both passed in.  Think about what behavior you want in each of these cases and write a "case" (look elsewhere in test_tools for how we are handling this now)